migration/migrator/data/course_tables.sql

```diff
@@ -283,6 +283,7 @@ CREATE TABLE public.gradeable (
     g_grade_released_date timestamp(6) with time zone NOT NULL,
     g_min_grading_group integer NOT NULL,
     g_syllabus_bucket character varying(255) NOT NULL,
+    g_allow_custom_marks boolean NOT NULL,
```

Comment:
This is missing the default value and the order doesn't match.

You should not manually make this edit, rather run the 'dump' command from step 3:
https://submitty.org/developer/migrations#writing-new-migrations